.github/workflows/Release.yaml

```
@@ -79,12 +79,20 @@ jobs:
     needs: [validate-push-release-request]
     outputs:
       gh-releases-matrix: ${{ steps.release-image.outputs.gh-releases-matrix }}
-      release-commit-sha: ${{ steps.release-commit-sha.outputs.release-commit-sha }}
     env:
       IS_PROD: ${{ ! startsWith(inputs.oci-image-name, 'mock-') }}
+      RELEASE_BRANCH: _releases
```

```suggestion
      RELEASES_BRANCH: _releases
```

to be consistent (also applicable to following occurrences)

---

Fixed in [3bc74db](https://github.com/canonical/oci-factory/pull/558/commits/3bc74db02725b848bf9839ef9d064938108b512c)